app/views/planning_applications/review/assessment_details/_form.html.erb

```diff
@@ -5,6 +5,7 @@
 ) do |form| %>
   <%= form.govuk_error_summary(presenter: ErrorPresenter) %>
   <% planning_application.application_type.assessment_details.each_with_index do |assessment_detail, index| %>
+    <% next if assessment_detail == "check_publicity" %>
```

Comment:
This is 🙈 as reviewing the check publicity step would massively bloat this PR even more 😬